packages/fxa-auth-server/lib/payments/paypal.ts

```diff
@@ -108,13 +166,37 @@ export class PayPalHelper {
     }
   }
 
-  public generateIdempotencyKey(invoiceId: string, paymentAttempt: number): string {
-    return invoiceId + '-' + paymentAttempt
+  /**
+   * Generate a PayPal idempotency key, used as the MSGSUBID on PayPal NVP
+   * API calls.
+   *
+   * @param invoiceId
+   * @param paymentAttempt
+   */
+  public generateIdempotencyKey(
+    invoiceId: string,
+    paymentAttempt: number
+  ): string {
+    return invoiceId + '-' + paymentAttempt;
```

Comment:
nit^2: I believe the cool kids are writing stuff like `${invoiceId}-${paymentAttempt}`.

Reply:
I shall strive to be cooler.